README.md

```diff
@@ -27,8 +27,5 @@ export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$UnicyclePlanner_DIR/lib
 export UnicyclePlanner_DIR=/path/where/you/installed/
 export DYLD_LIBRARY_PATH=$LD_LIBRARY_PATH:$UnicyclePlanner_DIR/lib
 ```
-## Basic Idea
-<img src="https://github.com/robotology/unicycle-footstep-planner/blob/master/images/planner.gif">
-
-## Implementation
-<img src="https://github.com/robotology/unicycle-footstep-planner/blob/master/images/unicycleWalking.gif">
+## Basic Idea & Implementation
+See https://github.com/robotology/unicycle-footstep-planner/issues/7
```

Comment:
Why are not using the old format but using https://user-images.githubusercontent.com/18591940/35742336-68cbf374-083a-11e8-8393-c0a5ea268866.gif and https://user-images.githubusercontent.com/18591940/35742348-71071cb2-083a-11e8-97a7-3577d997df8f.gif ?

Reply:
Those link are automatically generated when loading images in issues. Initially to solve the problem I inserted the images in the repository, and later I recognized it was actually a bad idea. Actually I did not think either to this solution when creating the issue! Thanks